sea-orm-macros/Cargo.toml

```diff
@@ -29,4 +29,7 @@ sea-orm = { path = "../", features = ["macros"] }
 serde = { version = "1.0", features = ["derive"] }
 
 [features]
+default = ["derive"]
```

Comment:
Why do we need a feature here? I think we always want to enable this right?

Reply:
Backward compatibility. Not a must, I just think it's better to ship with `derive` by default like it used to be.

Reply:
We have two features in `sea-orm-macros`
1. `derive` - provide all derive macros of SeaORM, e.g. `DeriveEntityModel`
2. `strum` - provide the `EnumIter` derive macros

I split the `sea-orm-macros` in half because one can opt-out the `derive` feature in SeaORM by specifying `sea-orm = { version = "*", default-features = false }`. Where the `sea-orm-macros` is a optional dependency. But now, the `strum` reside in it, making it a "compulsory" dependency. Instead, we turn on / off the `derive` feature as needed.

In contrast, we ALWAYS want the `strum` feature to be enabled.

Hence,

```diff
- sea-orm-macros = { version = "0.11.0", path = "sea-orm-macros", default-features = false, optional = true }
+ sea-orm-macros = { version = "0.11.0", path = "sea-orm-macros", default-features = false, features = ["strum"] }

[features]
- macros = ["sea-orm-macros", "sea-query/derive"]
+ macros = ["sea-orm-macros/derive", "sea-query/derive"]
```

Reply:
Umm. This sounds complicated. Well but let it be for now.